scorex-consensus/src/main/scala/scorex/consensus/nxt/NxtLikeConsensusModule.scala

```diff
@@ -10,11 +10,10 @@ import scorex.utils.{NTP, ScorexLogging}
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
-import scala.concurrent.duration._
 import scala.util.{Failure, Try}
 
 
-class NxtLikeConsensusModule(AvgDelay: Long = 5.seconds.toMillis)
+class NxtLikeConsensusModule(AvgDelayInSeconds: Long = 60)
```

Comment:
AvgDelay will pass from our Waves Application, we don't need to change its default value (5 sec) in Scorex